openflow15/nxt_message_test.go

```diff
@@ -0,0 +1,14 @@
+package openflow15
```

Comment:
Do we need to add copyright in new file?

Reply:
Shall we use a separate PR to add copyright for the project, Since many files in this project don't have the correct copyright.

Reply:
I don't think we should add copyright to the project. It's a cloned project, Antrea authors don't own most of the code.